examples/java/core/imports/src/main/java/com/microsoft/bond/examples/Imports.java

```diff
@@ -0,0 +1,41 @@
+package com.microsoft.bond.examples;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import com.microsoft.bond.Deserializer;
+import com.microsoft.bond.Serializer;
+
+import com.microsoft.bond.protocol.CompactBinaryReader;
+import com.microsoft.bond.protocol.CompactBinaryWriter;
+
+import com.microsoft.bond.examples.imports.Message;
+import com.microsoft.bond.examples.imports.common.Priority;
+
+public class Imports {
+
+    public static void main(final String[] args) throws IOException {
+
```

Comment:
I don't think there's much value in consistency on this particular point, but most of our code has the blank. IDEA has no opinion and its auto-format will leave it alone. Did a spot check of some top trending and starred Java repos and the blank beats !(the blank) 4-2.